skimage/transform/_geometric.py

```diff
@@ -281,28 +281,28 @@ class FundamentalMatrixTransform(_GeometricTransform):
 
     >>> tform_matrix.estimate(src, dst)
     True
-    >>> tform_matrix.params
-    array([[-0.2178588368,  0.4192819131, -0.0343074756],
-           [-0.0717941428,  0.0451643229,  0.0216072614],
-           [ 0.2480621133, -0.4294781423,  0.0221019139]])
+    >>> tform_matrix.params  # doctest: +FLOAT_CMP
```

Comment:
Again, can't this be a global doctest option?

Reply:
There's the [option in `setup.cfg`](https://github.com/scientific-python/pytest-doctestplus?tab=readme-ov-file#floating-point-comparison). Not fond of adding a `setup.cfg` again just for this.

I think the `+FLOAT_CMP` is actually not necessary. I added this before I discovered that another [test](https://github.com/scikit-image/scikit-image/pull/7288#discussion_r1492444288) was changing the precision for floating point comparisons. So I'll remove it in a follow-up PR. 

Though, in cases like this I would still prefer the more explicit local option unless we have a reason to to not use the default precision globally?

Reply:
https://github.com/scientific-python/pytest-doctestplus/issues/185

Reply:
A global config should be sufficient, but fine with having exceptions defined this way.

Reply:
For completeness, `saimn` pointed out in https://github.com/scientific-python/pytest-doctestplus/issues/185 that pyproject.toml is supported since https://github.com/scientific-python/pytest-doctestplus/pull/222